driver/conv_driver.hpp

```diff
@@ -125,6 +125,26 @@ struct AutoMiopenWarmupMode
     bool debug_find_enforce_disable_prev;
 };
 
+struct AutoConvDirectNaiveAlwaysEnable
+{
+    AutoConvDirectNaiveAlwaysEnable()
+    {
+        debug_always_enable_conv_direct_naive_prev = miopen::debug::AlwaysEnableConvDirectNaive;
+        miopen::debug::AlwaysEnableConvDirectNaive = true;
+    }
+    AutoConvDirectNaiveAlwaysEnable(const AutoConvDirectNaiveAlwaysEnable&) = delete;
+    AutoConvDirectNaiveAlwaysEnable(AutoConvDirectNaiveAlwaysEnable&&)      = delete;
+    AutoConvDirectNaiveAlwaysEnable& operator=(const AutoConvDirectNaiveAlwaysEnable&) = delete;
+    AutoConvDirectNaiveAlwaysEnable& operator=(AutoConvDirectNaiveAlwaysEnable&&) = delete;
```

Comment:
[Recommendation] Great! But are you really sure we need this kind of protection for such a simple object? I would remove this.

Reply:
OK, let me remove these lines.